model/database/registry.yaml

```diff
@@ -461,11 +461,11 @@ groups:
         type:
           members:
             - id: gateway
-              value: "gateway"
+              value: "Gateway"
```

Comment:
this is .NET casing, Java uses GATEWAY and DIRECT and there is no REST API property - let's keep values snake_case at least when there is no standard cross-language name for them